apps/deploy-web/src/components/sdl/PlainVMForm.tsx

```diff
@@ -0,0 +1,5 @@
+"use client";
+
+export const PlainVMForm: React.FunctionComponent = () => {
+  return <>Comming Soon</>;
```

Comment:
@anilmurty Do you want to release it like this?

Reply:
If it's going on the main deploy page, then I think no. This task was to just merge the upload SDL into the "create your template" tile. The VM tile can be done separately

Reply:
so should we set this thread to resolved status.

Reply:
I have commented out the "Plain Linux" tile for now.